controllers/get-article-comments.controller.js

```diff
@@ -0,0 +1,13 @@
+const { selectArticleComments } = require("../models/topics.models")
+
+const getArticleComments = (req,res)=>{
+    const {article_id} = req.params
+    if(!Number(article_id)){
+        return res.status(400).send({msg:"Article ID must be a number"})
+    }
+    selectArticleComments(article_id).then((comments) => {
+    res.status(200).send(comments)
+})
```

Comment:
Nice but we should be sending an object back here with an appropriate key to hold this data

We should also have a .catch(...) here to deal with any errors